docs/current/742989-cookbook.md

```diff
@@ -652,6 +652,31 @@ The following code listing uses a cache volume to persist a service's data acros
 
 [Learn more](./guides/757394-use-service-containers.md)
 
+### Add Multiple Environment Variables to a Container
+
+The following code listing demonstrates how to add multiple environment variables to a container.
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go file=./guides/snippets/use-services/environment-variables/main.go
```

Comment:
Is this code used within the `use services` guide? If not ie. it is standalone code for the cookbook, it should be placed in `cookbook/snippets`.

Reply:
The correct location is `cookbook/snippets` not `guides/snippets`. See above and https://github.com/dagger/dagger/tree/main/docs/current/cookbook/snippets

Reply:
You're right, fixed 🙏